pkg/apiserver/authorization/rbac/rbac.go

```diff
@@ -250,11 +250,11 @@ func (r *RBACAuthorizer) visitRulesFor(requestAttributes authorizer.Attributes,
 				}
 			}
 		} else if requestAttributes.GetResourceScope() == request.DevOpsScope {
-			if workspace, err = r.am.GetDevOpsControlledWorkspace(requestAttributes.GetDevOps()); err != nil {
-				if !visitor(nil, "", nil, err) {
-					return
-				}
-			}
+			//if workspace, err = r.am.GetDevOpsControlledWorkspace(requestAttributes.GetDevOps()); err != nil {
```

Comment:
Why do we keep those unused lines?

Reply:
Currently, we didn't move this part to any projects. And there's no good solution to solve it. In order to avoid forgetting it, I recommend leaving it until we have a solution for it.